widgetastic_manageiq/__init__.py

```diff
@@ -4274,12 +4274,37 @@ def schedule_migration(self, plan_name, cancel=False, after_mins=1):
         try:
             el = self._get_plan_element(plan_name)
             schedule_button = self.browser.element(self.ITEM_SCHEDULE_BUTTON_LOCATOR, parent=el)
```

Comment:
Optional:
I would use a `widget` inside a `widget` for such things.

Reply:
`Button` ? I think I tried that and for some reason did not work that I can't recall.

Reply:
Can you point me to an example please?

Reply:
@kedark3 you can refer wt.core or wt.pfy you will get lots of examples.